docs/v2/demos/selected-days.html

```diff
@@ -12,15 +12,22 @@
 
         <script type="module">
             import HelloWeek from './scripts/hello.week.min.js';
+          
+            const date = new Date();
+            const currentYear = date.getFullYear();
+            let currentMonth = date.getMonth() + 1;
+            currentMonth = currentMonth < 10 ? '0' + currentMonth : currentMonth;
```

Comment:
Thanks for you contribution, can you simplify: 

```suggestion
            currentMonth.toString().padStart(2, 0);
```